eureka2-write-server/src/main/java/com/netflix/eureka2/server/service/overrides/OverridesModule.java

```diff
@@ -10,10 +11,10 @@
 public class OverridesModule extends AbstractModule {
     @Override
     protected void configure() {
+        // override this binding with more specific impls in non-local-testing environments
+        bind(LoadingOverridesRegistry.ExternalOverridesSource.class).to(InMemoryOverridesSource.class).in(Scopes.SINGLETON);
 
         bind(OverridesService.class).to(OverridesServiceImpl.class);
-
-        // override this binding with more specific impls in non-local-testing environments
-        bind(OverridesRegistry.class).to(InMemoryOverridesRegistry.class);
+        bind(OverridesRegistry.class).to(LoadingOverridesRegistry.class);
```

Comment:
Since it is a very specific overrides setup, maybe the module should be renamed to InMemoryOverridesModule? Or we could provide alternatively a builder for the module itself.